property-files-loader/src/main/java/com/msm/property/file/loader/readers/PropertyFilesReader.java

```diff
@@ -0,0 +1,90 @@
+package com.msm.property.file.loader.readers;
+
+import com.msm.property.file.loader.annotations.PropertiesResource;
+import com.msm.property.file.loader.annotations.PropertyName;
+import org.reflections.Reflections;
+
+import java.io.*;
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author riste.jovanoski
+ * @since 6/13/2017
+ */
+public class PropertyFilesReader {
```

Comment:
The class looks confusing. You should separate the file loading logic and properties loading logic in a different classes.